tools/sr_bowtie/sRbowtie.xml

```diff
@@ -14,7 +14,7 @@
         #end if
         #if $input.extension == "fasta":
             #set format = "-f"
-        #elif $input.extension == "fastq":
+        #elif $input.extension == "fastq" or $input.extension == "fastqsanger":
```

Comment:
you're probably looking for `$input.is_of_type('fastq')` as briefly mentioned in https://docs.galaxyproject.org/en/master/dev/schema.html#id4

Reply:
Yes it is exactly what I was looking for ! Thanks @mvdbeek !